### ISSUE TITLE
URGENT New members via website, not registering in Chargify or Capsule but have ID and Member Directory entry

### ISSUE THREAD

**Issue by Juliehitgub** [theodi/shared#917]
We have three new members register in March via our website last week. Capsule ID refs are WD3322WJ, NT2408EK, DM9497UL.
This ID ref is NOT recognised in Capsule and they have not been charged for membership in Chargify but they all have a Member Directory entry.
Can someone have a look, is this the same as problem reported a while back? 
This needs to be fixed to allow me to process membership, add them to our count, and also to obtain payment.

**Comment by pezholio** [comment]
This is fixed - we're going to have a look at ways in which we can get alerts when things seem to be going wrong so we can fix problems like this before you notice :)

**Comment by Juliehitgub** [comment]
The Capsule entries for these 3 organisations have came through but nothing
has appeared so far in Chargify, despite refreshing a few times.

Also, new member signed up today just before stand up, they are not in
Capsule as member but are in Chargify and Member Directory. ID ref from
email is: LT0735MW, Map of Agriculture.

Julie McMahon, Membership Administrator
+44 (0) 20 3598 9395

The Open Data Institute, 65 Clifton Street, London EC2A 4JE
<http://www.theodi.org/>

**Comment by Floppy** [comment]
Reopened to check Chargify status.

**Comment by pezholio** [comment]
I wouldn't have thought Chargify had anything to do with the queues?

**Comment by Floppy** [comment]
no, it doesn't, hence I'm wondering why they're not there. I'll take a look.

**Comment by Floppy** [comment]
Charify stuff is queued sometimes, but not during organisation signup, as they'd have to enter their payment details. So it's odd.

**Comment by pezholio** [comment]
Have they definitely paid by credit card? They could be paying by Invoice - maybe?

**Comment by Juliehitgub** [comment]
Stu, Steph to double checking Stripe/ Xero etc so she usually lets me know
when payments appear so I assumed credit card. Will update when I get the
info.

Julie

Julie McMahon, Membership Administrator
+44 (0) 20 3598 9395

The Open Data Institute, 65 Clifton Street, London EC2A 4JE
<http://www.theodi.org/>

**Comment by Juliehitgub** [comment]
Steph has no record of them on her system requesting invoices, so no
payment has been received or invoice has been sent in respect of these
memberships.



Julie McMahon, Membership Administrator
+44 (0) 20 3598 9395

The Open Data Institute, 65 Clifton Street, London EC2A 4JE
<http://www.theodi.org/>

**Comment by pezholio** [comment]
Hey Julie, can you check if they requested an invoice please? Chargify seems to be working OK, but if they requested an invoice payment, there's a risk that this might've been lost. It's easily regenerated if that's the case.

**Comment by Juliehitgub** [comment]
Have checked, we've got no record of them requesting invoice. If easy for
you to find this request that would be great or I can ask finance to issue
invoice today?




Julie McMahon, Membership Administrator
+44 (0) 20 3598 9395

The Open Data Institute, 65 Clifton Street, London EC2A 4JE
<http://www.theodi.org/>

**Comment by pezholio** [comment]
Hey Julie

Sorry, was working with a contractor all day today, so not had sight of my
emails. Will have a look tomorrow :)
